### PR TITLE
Removed v0 convolution and group convolution

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

**Comment by ilyachur** [comment]
> I'm not sure if we really need to move these v0 conv ops into test directory.
> Can we just remove it?
> If you want to keep it here please create a ticket to clean-up it in the future.
> 
> Its better to keep old code in the version control instead of adding garbage to the test folder IMHO.

I moved v0 convolution to test folder because at the current moment it is used in the Interpreter plugin.
As I know @mikhail-treskin already make some experiments to migrate Interpreter from opset0 to opset1.

File: ngraph/src/ngraph/builder/split.hpp [hunk 1]
```diff
@@ -49,6 +49,7 @@ namespace ngraph
         ///
         /// \return     The vector containing multiple outputs we split input node into.
         ///
+        NGRAPH_API
```
Comment (GlebKazantaev) [inline ngraph/src/ngraph/builder/split.hpp:52]: why we expose this builders? I think we always suggest to use operations from opsets

File: ngraph/test/backend/convolution.in.cpp [hunk 2]
```diff
@@ -37,20 +38,20 @@ NGRAPH_TEST(${BACKEND_NAME}, convolution_outlining)
...
-                                              CoordinateDiff{0, 0},
-                                              Strides{1, 1});
-    auto conv2 = make_shared<op::Convolution>(conv1,
-                                              B,
-                                              Strides{1, 1},
-                                              Strides{1, 1},
-                                              CoordinateDiff{0, 0},
-                                              CoordinateDiff{0, 0},
-                                              Strides{1, 1});
+    auto conv1 = make_shared<op::v0::Convolution>(A,
```
Comment (GlebKazantaev) [inline ngraph/test/backend/convolution.in.cpp:41]: I thought you removed v0 namespace

File: ngraph/src/ngraph/builder/split.hpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/src/ngraph/builder/split.hpp:52]: It uses in group_convolution which I moved to tests.
I have a plan to remove this builders for v0 ops.

**Pull request merged**